modules/apps/frontend-js/frontend-js-a11y-web/build.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+	compileOnly group: "biz.aQute.bnd", name: "biz.aQute.bnd.annotation", version: "4.2.0"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
+	compileOnly project(":apps:frontend-js:frontend-js-loader-modules-extender-api")
+	compileOnly project(":apps:static:portal-configuration:portal-configuration-metatype-api")
+}
+
+jar {
+	enabled System.getenv('NODE_ENV') == "development"
+}
```

Comment:
Just an observation about this implementation, this won't respect `nodejs.node.env` when we're tinkering with this module to deploy, so if you come to work on this module you need to set it to `NODE_ENV=development` in the console before calling the deploy.

Reply:
In addition to the bare env var, maybe we can check if the property was set via `build.<username>.properties `as well. I'm not sure this is a common pattern but it makes sense to me.

Reply:
Hmm, I think it's a good idea, but I have no idea how to implement this 😅, well, I can do some research and see what I can do

Reply:
I haven't done it either but as a last resort you can read the file directly. I would guess that, being a properties file, it should be available somewhere like in project.properties: https://github.com/brianchandotcom/liferay-portal/blob/ea7567225b6382054d1128a0d0c82146b1acaa56/modules/releng.gradle#L116

Reply:
Hmm thank you @nhpatt, this will help

Reply:
I did some testing getting the `project.properties` property and it seems to work fine.